### PR TITLE
Alter link text

### PR DESCRIPTION
**Pull request by genjohnson**
The difference between where to go to manage user accounts and manage Person nodes can be confusing for users because of the title of the links - "People" to manage user accounts and "Person" for the content type plus the default "Person" menu link to the page `/people`.

This pull request attempts to reduce this confusion by changing the title of the "People" link (to manage user accounts) to "Users".

### CODE REVIEW

**Comment by brandonneil** [comment]
_TLDR: Sorry, this got long. I think we should do usability testing. If an identified issue, propose issue for 8.3.x core, and "backport" to standard and custom service._

My only hesitation doing this is that since this is a core-provided menu text, we will be maintaining this change forever, unless we fix it in core as well. [Core's update policy ](https://www.drupal.org/core/d8-allowed-changes) _does_ allow for changing of strings in minor versions, so yay:

> Minor releases provide new improvements and functionality without breaking backward compatibility (BC) for public APIs. The following types of changes are allowed for minor releases in addition to those allowed for patch releases. 

>Functionality and frontend new features
changes to behavior that existing sites might rely on
**_string, user interface, or render array changes_**
CSS, markup, or template changes (except for stable base themes)

Also, this would/should be changed in the custom service as well for consistency.

I cannot find an issue about the usability of the "People" menu in the core issue queue. I imagine this is only an issue for sites that have a person-based content entity and user entity, so it never comes up in the normal Drupal core usability studies they do each year at UMN.

For our service, it seems like this would only affect users who can manage people as well as manage content, so it would be webmasters only.

Perhaps it is time we do some usability testing on the standard service as we look forward to Drupal 8?

I think if we wanted to truly attempt to propose a fix to this in core we would need to:

- Do usability testing and show that this is an issue for sites with people-based content and users who manage both people content and user accounts.
- Create a new issue against the next minor core version (as of now 8.3.x) with our usability findings. And propose a new name (Users, User accounts, etc).

Alternately, perhaps renaming the Standard Service's Person content type to Person Profile and providing more help text about the admin/people page could help address this issue?

**Comment by genjohnson** [comment]
This needs more discussion, so for now I'm going to mark as "postponed"

**Comment by genjohnson** [comment]
On second thought, this is a very minor change to make if we decide to do it in the future, so marking as "won't fix" and closing.
